src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php

```diff
@@ -87,7 +88,15 @@ public function configureOptions(OptionsResolver $resolver): void
             'csrf_message' => 'The CSRF token is invalid. Please try to resubmit the form.',
             'csrf_token_manager' => $this->defaultTokenManager,
             'csrf_token_id' => null,
+            'csrf_data_attr' => 'csrf-protection',
```

Comment:
If it changes, the controller.js will stop working. Is it really necessary to make it configurable? I wonder if setting a "unique" attribute such as `sf-csrf-protection` would be enough...

Reply:
Right. I removed this option and moved it to a constructor argument so that it's still possible to change it, but it's not exposed.